lib/aerosol/instance.rb

```diff
@@ -44,8 +46,17 @@ def self.request_all
 private
   def describe!
     ensure_present! :id
+    attempts ||= 0
     result = Aerosol::AWS.compute.describe_instances('instance-id' => id).body
     result['reservationSet'].first['instancesSet'].first rescue nil
+  rescue Fog::Compute::AWS::NotFound => ex
+    if attempts < 3
+      attempts += 1
+      logger.error "#{ex.class}: #{ex.message} - retrying"
+      retry
```

Comment:
That's probably a good idea.  Not sure how long of a wait it should be.  Just 1 second?